samples/js/browser/DialogServiceConnector/README.md

```diff
@@ -0,0 +1,28 @@
+# JavaScript voice assistants sample for web browsers  
+
+This sample demonstrates how to connect to and interact with your voice assistant using the Speech SDK for JavaScript on a web browser, like Microsoft Edge, or Chrome.
+See [this page](https://github.com/microsoft/cognitive-services-speech-sdk-js) for introductory information on the Speech SDK for JavaScript.
```

Comment:
Are we allowing to put page tags in URLs here? If so, this should probably be https://github.com/microsoft/cognitive-services-speech-sdk-js#readme